### PR TITLE
Drupal 6 part 3

### PR DESCRIPTION
**Pull request by chrisarusso**

### CODE REVIEW

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+ + risk tolerance
+ + likelihood to be targeted
+ + budget to improve your website
+
+
+##### Considerations to _not_ upgrade
+
+Choosing _not_ to upgrade your website is typically dictated by your organization having neither internal
+technical expertise or sufficient funds to hire a partner. When this is the case, if a small budget can be
+afforded, a site audit is usually the next best option to have some peace of mind that you're taking the measures your're able to
```
Comment (AnneTee) [inline _posts/2016-01-25-drupal-6-part-3.md:89]: @chrisarusso typo in "you're"

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  like the [static site generator Jekyll](https://jekyllrb.com/) from which our own
+   web site is built. An honest consultant will tell you when Drupal is a good fit
+   and when it's not.
+   
+   
+ But alas, if you're determined to stick with Drupal (as you typically should be)
+ and you'd like to upgrade, the final consideration 
+ is what platform to upgrade to, Drupal 7 or 8? Ah-ah-ah...
+
+<img src="/assets/img/ah-ah-ah-terminal.jpg" alt="ah ah ah">
```
Comment (AnneTee) [inline _posts/2016-01-25-drupal-6-part-3.md:152]: @chrisarusso ಠ_ಠ

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+One other consideration independent
+of your specific website configuration is your server autonomy dictated by your hosting provider. Running old software is
+usually tolerated in a shared hosting environment for only so long before upgrades
+are enforced. Warning emails are often (not always) sent out in advance of server-level upgrades, however,
+ it's easy to miss those emails, and it hurts to be caught by surprise that some PHP version upgrade renders
+your site completely useless. Understanding the level of control you as the 
+site owner has over your server is crucial.
+
+    
+##### Considerations in upgrading
```
Comment (flowersw) [inline _posts/2016-01-25-drupal-6-part-3.md:108]: Consider making this phrase "Considerations in upgrading" similar to the title above "Considerations to not upgrade". Maybe change the above phrase to "Considerations in not upgrading" @chrisarusso

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+##### Considerations to _not_ upgrade
+
+Choosing _not_ to upgrade your website is typically dictated by your organization having neither internal
+technical expertise or sufficient funds to hire a partner. When this is the case, if a small budget can be
+afforded, a site audit is usually the next best option to have some peace of mind that you're taking the measures your're able to
+ mitigate the risk of running outdated software. A group like Savas Labs can perform a 
+thorough assessment of risk and make budget-conscious recommendations
+on how to best harden the site against outside threats without doing the more costly
+full-site upgrade. Primary attack vectors that we focus lie in custom code, and
```
Comment (kostajh) [inline _posts/2016-01-25-drupal-6-part-3.md:93]: `that we focus on`

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+ + likelihood to be targeted
+ + budget to improve your website
+
+
+##### Considerations to _not_ upgrade
+
+Choosing _not_ to upgrade your website is typically dictated by your organization having neither internal
+technical expertise or sufficient funds to hire a partner. When this is the case, if a small budget can be
+afforded, a site audit is usually the next best option to have some peace of mind that you're taking the measures your're able to
+ mitigate the risk of running outdated software. A group like Savas Labs can perform a
```
Comment (kostajh) [inline _posts/2016-01-25-drupal-6-part-3.md:90]: `A group like Savas Labs`, maybe just say "A Drupal service provider (such as ourselves)"

File: _posts/2016-01-25-drupal-6-part-3.md (hunk 3 above)
Comment (kostajh) [inline _posts/2016-01-25-drupal-6-part-3.md:108]: You might also break up this section into a few sub-sections, although I see you're already in h5 territory

File: _posts/2016-01-25-drupal-6-part-3.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+Though some advisers (solicited or not) might tell you otherwise, you _always_ have options as to what to do
+with your website if you're running Drupal. One of them is of course nothing, and the others require more thought.
+
+What is the right thing to do can be a hot-button topic. Some technologists are purists, 
+and believe it sacrosanct to even consider running software beyond EOL. However, we appreciate
+that although ideally site owners would always have been prepared and budgeted for 
+software upgrades, sometimes life happens, and other business priorities and budget
+availability don't perfectly align with timing for a software upgrade.
+
```
Comment (dmurphy1) [inline _posts/2016-01-25-drupal-6-part-3.md:39]: @chrisarusso - I thought this sentence "However, we appreciate..." was a little lengthy and difficult to follow. Maybe there's a better way of wording it?

**Pull request merged**